src/sentry/scripts/digests/digests.lua

```diff
@@ -9,11 +9,11 @@ function table.extend(t, items, length)
     end
 end
 
-function table.slice(t, ...)
-    local start, stop = ...
+function table.slice(t, start, stop)
```

Comment:
I tacked this change on here also because this signature is/was kind of bizarre. I think I had planned to make this more clever but never needed to use this in any way other than to chop off the head of a list and return the remaining elements.